backend/utils/tests/test_cors_origin.py

```
@@ -0,0 +1,182 @@
+        assert ro != None  # noqa: E711
+        assert ro != 42
+        assert ro != []
+
+    def test_unhashable(self):
+        """``__hash__ = None`` prevents the equality/hash contract from being
+        violated if anyone wraps the allow-list in a ``set``/``frozenset``."""
+        ro = RegexOrigin(r"^x$")
+        with pytest.raises(TypeError):
+            hash(ro)
+        with pytest.raises(TypeError):
+            {ro}  # noqa: B015
```

Ruff (0.15.12)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)